README.md

```diff
@@ -1 +1,109 @@
-# node-modules-json
\ No newline at end of file
+# Importing JSON files
+
+## Overview
+
+JSON is a widely use data format. When it comes to Node applications, developers like to use JSON file to store various data such as:
+
+* Configurations
+* Seed data (initial data to populate the development or test database)
+* Settings
+* Plugins for frameworks
+
+Let's implement code which reads a JSON file and parses the file data into JavaScript object (`read.js`). The `customers.json` has information about customers including their emails, balance, phone numbers and addresses. Please don't share this information with anyone.
```

Comment:
"Please don't this information with anyone" is ominous -- this is fake data, right? Because this repo is public and it's all already out there.

Assuming that it's fake data, asking students not to share it is unnecessary.